src/main/resources/sql/select/vdicontrol/availableuserdatasets/by-user-access.sql

```diff
@@ -17,4 +17,4 @@ JOIN
 ON
 	a.user_dataset_id = d.dataset_id
 WHERE
-	a.user_id = ? or a.is_public = 1
+	a.user_id = ? or a.is_public = '1'
```

Comment:
Curious about this one.  If in Oracle it's a varchar(1), then how did it work before?  And if numeric (Oracle's boolean) shouldn't this break?  Based on the name I feel like we should use Platform's boolean type handlers.

Reply:
In Oracle, this gets treated as a number and in Postgres it's treated as a boolean. I think there's also a DBPlatform method for this which I discovered later.